mobile-version.html

```diff
@@ -0,0 +1,124 @@
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javascript</a></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <div class="job-4">
+                <img src="/images/Snapshoot-Portfolio-2.jpg" alt="job-4 image">
+                <div class="content">
+                    <h3>Multi-Post Stories</h3>
+                    <ul>
+                        <li>Canopy</li>
+                        <li>Back End DEv</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javascript</a></li>
+                    </ul>
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+        </section>
+        <section class="About">
+            <h2>About <br> Myself</h2>
+            <p class ="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a href="#" class="connect">LET'S CONNECT</a>
+                <div class="contacts">
+                    <ul>
+                        <li><a href="#" class ="link"><i class="fa fa-twitter"></i></a>
+                        <li><a href="#" class ="link"><i class="fa fa-linkedin-square" ></i></a></li>
+                        <li><a href="#" class ="link"><i class="fa fa-github"></i></a></li>
+                        <li><a href="#" class ="link"><i class= "fa fa-hand-peace-o"></i></a></li>
+                        <li><a href="" class="link"><i class="material-icons">feedback</i></a></li>
+                    </ul>
+                </div>
+                <button type="button">Get Resume</button>
+                <div class="skills">
+                    <p class="skill"> Languages</p>
+                    <i class="fa-light fa-greater-than"></i>
+                    <ul class = "languages">
+                        <li><img src="/images/js.jpg" alt=""><a href="#" class= "tech" >JavaScript</a></li>
+                        <li><img src="/images/html.jpg" alt=""><a href="#" class="tech">HTML5</a></li>
+                        <li><img src="/images/css.jpg" alt=""><a href="#" class= "tech">CSS</a></li>
+                    </ul>
+
+                    <p class="skill">Frameworks</p>
+                    <i class="fa-light fa-greater-than"></i>
+                    <hr>
+                    <p class="skill">Skills</p>
+                    <i class="fa-light fa-greater-than"></i>
+                    <hr>
+
+                </div>
```

Comment:
- [ ] As I can see here for this section the padding and margin and also the alignments for the items so please Fix them to be the same as the template and add the arrow in the side also.
  - your design 
   
![langs](https://user-images.githubusercontent.com/58553711/177006327-b10521e2-baa8-4284-b01d-afe27f453c01.PNG)

- Figma design
   
![image](https://user-images.githubusercontent.com/58553711/177006351-f322821b-bc91-4fef-ad86-d7b310b027a4.png)


- [ ] Kindly change the color for the `p` tag to be the same as the design. 🙏🏻🙏🏻